django/forms/fields.py

```diff
@@ -734,6 +734,15 @@ class NullBooleanField(BooleanField):
     """
     widget = NullBooleanSelect
 
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
+        if not isinstance(self.widget, NullBooleanSelect):
+            self.choices = self.widget.choices = [
```

Comment:
The assignment to `self.choices` has no effect. Not being a subclass of `ChoiceField`, it's not used in validation. (Nothing fails removing it.)